flex/validation/common.py

```diff
@@ -299,7 +300,7 @@ def generate_enum_validator(enum, **kwargs):
 
 
 @skip_if_empty
-def validate_allof_anyof(value, sub_schemas, context, method, **kwargs):
+def validate_composition(value, sub_schemas, context, method, default_error_message=None, **kwargs):
```

Comment:
I think `error_message_override` might be a more appropriate name than `default_error_message`.  Alternatively I'd support removing this in place of just writing 3 small functions which each raised the appropriate error message for `all/any/exactly_one_of` so that this function doesn't have to expand it's signature.  Your call.